### PR TITLE
Update SCREAM compsets

### PR DESCRIPTION
**Pull request by brhillman**
Update SCREAM compsets and our ultra-high-resolution configurations (ne512 and ne1024). This adds two new compsets: FSCREAM-LR-DYAMOND1 and FSCREAM-HR-DYAMOND1, which properly set the run start date and initial conditions for hindcast simulations for the DYAMOND1 period (initialized 2016 August 01). Initial conditions are also added for ne4 and ne30, so these lower-resolution grids can be used to test these compsets as well. Defaults are also updated for ne512 and ne1024 (for any compset), and dycore settings for FSCREAM-HR ne4 and ne30 are fixed so these resolutions should run with these compsets now as well.

### CODE REVIEW

**Comment by brhillman** [comment]
Also, question: right now, we the FSCREAM-HR compsets use iradsw=iradlw=1, which means do radiation every timestep. This is probably excessive for most resolutions, especially for testing. For ne1024, right now we are setting this to be 4, which equates to every 5 minutes with a 75s physics timestep (the default for ne1024). Making this the default though will affect other resolutions, like ne4 which uses a default ATM_NCPL=12, so a value of 4 means that radiation would only be updated every 8 hours, every 2 hours for ne30 which uses ATM_NCPL=48. The built-in default is -1, which means every hour for each resolution. Should we just remove the setting of this from the use case file for FSCREAM-HR?

File: components/cam/cime_config/config_component.xml [hunk 1]
```diff
@@ -363,8 +363,10 @@
     <desc compset="_CAM5%MG1M"   >MG1.0 w/ modified activation:</desc>
     <desc compset="_CAM5%UNI"    >UNICON (modified mg1.0):</desc>
     <desc compset="CAM[45].*_BGC%B">CAM prognostic CO2 cycle turned on.</desc>
-    <desc compset="_CAM5.*SCREAM-LR"   >Low Resolution Fortran version of SCREAM with SHOC, P3, and RRTMG.  All other aspects of this run are identical to E3SMv1</desc>
-    <desc compset="_CAM5.*SCREAM-HR"   >Fortran version of SCREAM with NH dycore, SHOC, P3, RRTMGP, and no deep convection scheme called.</desc> 
+    <desc compset="_CAM5.*SCREAM-LR-DYAMOND1">Fortran version of SCREAM with SHOC, P3, and RRTMGP. Initialized for DYAMOND1 (2016-08-01).</desc>
```
Comment (bogensch) [inline components/cam/cime_config/config_component.xml]: Maybe we should note in each desc for all SCREAM compsets that prescribed aerosols are used?

Comment (brhillman) [reply components/cam/cime_config/config_component.xml]: Oh yeah, good idea. I'll push a commit.

Comment (brhillman) [reply components/cam/cime_config/config_component.xml]: Done.

**Comment by bartgol** [comment]
@brhillman Isn't that parameter configurable from the input namelists?

**Comment by brhillman** [comment]
@bartgol you mean the radiation timestep? It is, we set it via user_nl_cam in our ne1024 run scripts right now. The default in E3SM is just 1 hour. I think we added a default of 1 timestep when we were only using this with ne1024 and thought we should start with every timestep.

**Comment by bogensch** [comment]
I think it makes sense to remove it, @brhillman in light of your comments and the evolution of the compsets

**Comment by brhillman** [comment]
Actually I guess we could conditionally set it based on the value of hgrid:
```
<iradsw hgrid="ne1024np4"> 4 </iradsw>
```
should do it?

**Comment by PeterCaldwell** [comment]
About radiation call frequency: yeah, it seems like setting it to -1 in general, then overriding with 4 for ne1024 (and perhaps ne512, etc) is the way to go. At some point I'd like to actually do convergence studies with rad frequency and other things to decide more objectively what to set them at.

**Comment by brhillman** [comment]
@PeterCaldwell yeah, I agree, would love to have the opportunity to do that.

**Comment by brhillman** [comment]
Okay, setting `iradsw=iradlw=4` conditional on hgrid works, so I made that change, and leave the default -1 for all other resolutions for now. I also fixed the setting of `se_ftype`, so we only set that to 1 now when using ne1024 (and do that for both FSCREAM-HR and FSCREAM-LR families of resolutions, in case we ever wanted to use ne1024 with FSCREAM-LR).

**Comment by PeterCaldwell** [comment]
I notice AV1C04-P2 is mentioned e.g. in "components/cam/bld/namelist_files/use_cases/2000_scream_hr_av1c-04p2.xml". Do we still use anything from the AV1C compset? If not, why not delete this stuff? If we do use AV1C stuff, we should switch to F2010_CMIP6 compsets, which use more modern input data.

File: components/cam/cime_config/config_component.xml [hunk 2]
```diff
@@ -27,15 +27,15 @@
   </entry>
 
   <entry id="CAM_TARGET">
-         <type>char</type>
-         <valid_values>preqx,preqx_kokkos,preqx_acc,theta-l</valid_values>
-         <default_value>preqx</default_value>
-	 <values>
+    <type>char</type>
+    <valid_values>preqx,preqx_kokkos,preqx_acc,theta-l</valid_values>
+    <default_value>preqx</default_value>
```
Comment (PeterCaldwell) [inline components/cam/cime_config/config_component.xml]: Do we really want preqx to be the default here?

Comment (brhillman) [reply components/cam/cime_config/config_component.xml]: I think we inherit that from E3SM. We can change it.

**Comment by PeterCaldwell** [comment]
I didn't notice whether the new SST/sea ice files that @crterai  is working on are included here or not. If not, I assume one of you will add them in a future PR?

**Comment by brhillman** [comment]
@PeterCaldwell right, I think Chris is still working on the SST files. We found some weird things earlier today involving missing data, so I think he's working on that. We will update the compsets when he gets that sorted out in a future PR. Regarding the mention of av1c in the use case names, I'm good with removing those. I'm assuming that @bogensch included those strings because some of the tunings were probably pulled from those compsets?

**Comment by brhillman** [comment]
@PeterCaldwell so here's the bigger question. Right now, we are making these compset definitions:
```
  <compset>
    <alias>FSCREAM-LR</alias>
    <lname>2000_CAM5%SCREAM-LR_CLM45%SPBC_CICE%PRES_DOCN%DOM_SROF_SGLC_SWAV</lname>
  </compset>

  <compset>
    <alias>FSCREAM-HR</alias>
    <lname>2000_CAM5%SCREAM-HR_CLM45%SPBC_CICE%PRES_DOCN%DOM_SROF_SGLC_SWAV</lname>
  </compset>
```
So those should be inheriting everything from the F2000 compsets. You think it's better to switch these to F2010? I think that would mean changing `2000_CAM5%` to `2010_CAM5%`, if that's the correct naming convention.

**Comment by PeterCaldwell** [comment]
@brhillman - yeah, switching F2000 to F2010 is the main thing I was suggesting (rather than just wanting the names to be changed). I was thinking this would be involved enough to warrant its own PR, but now that I think about it, probably just switching the long name (like you've modeled above) will fix the problem.

**Comment by brhillman** [comment]
@PeterCaldwell I'm not seeing many compsets named with "2010"...the only one I see is " F2010SC5-CMIP6". Is there a different name for this? Or do we somehow just not have them in our repo?

**Comment by PeterCaldwell** [comment]
I think this line is the one you'd want to copy the inheritance from: 

      <value compset="2010S_CAM5.*CMIP6"  >2010S_cam5_CMIP6</value>

**Comment by PeterCaldwell** [comment]
Sorry, I commented at the same time as you asked your question. yeah, I think there's only just that one F2010 compset, but it is the one with the more up-to-date inputdata. We should probably check that scream runs with it (though I can't imagine why it wouldn't).

**Comment by brhillman** [comment]
Cool, I'll try it out and make the change if it runs.

**Comment by bogensch** [comment]
Yes, the av1c-04p2 name is on the SCREAM compsets because the original tunings were based on these compsets.  I don’t see there being any reason to keep this naming convention anymore as we diverge from that compset.

**Comment by brhillman** [comment]
So looking at it a little more closely, it looks like all that stuff is just set in the use case file anyways. So maybe the thing to do is to update the settings in our SCREAM use case files to be consistent with the 2010 settings?

**Comment by PeterCaldwell** [comment]
Yeah, making the use case use 2010 stuff is the way to go. It seems like this 2010 change is getting to be more work than we might want to do in this PR. Save if for a future PR? Up to you.

**Comment by brhillman** [comment]
@PeterCaldwell I'm leaning towards making those changes in a separate PR, just to keep changes somewhat specific and isolated, and let folks like yourself and @bogensch look at just those settings alone without all this extra grid-specific stuff to make sure I don't screw anything up.

**Comment by bogensch** [comment]
Sounds smart to me

**Comment by PeterCaldwell** [comment]
Yeah, I like that better too. ;-)

**Comment by brhillman** [comment]
Great, thanks. @jgfouca pinging you for a review on this because I touched some stuff that lives in CIME. @mt5555 pinging you because I set some dycore defaults. @AaronDonahue this might affect simulations you do with the SCREAM compsets (it will change the radiation frequency). And @crterai you'll care about this too I'm sure.

**Comment by brhillman** [comment]
Pinging @mt5555 @crterai @AaronDonahue for review

**Pull request merged**